selenium/exercise6/Exer.rb

```diff
@@ -0,0 +1,16 @@
+require "selenium-webdriver"
+
+Selenium::WebDriver::Chrome::Service.driver_path = "C:/Users/naika/OneDrive/Documents/Drivers/chromedriver.exe"
+
+driver = Selenium::WebDriver.for :chrome
+
+driver.get "https://letcode.in/forms"
+
+wait = Selenium::WebDriver::Wait.new(:timeout => 10)
+
+dropdown_list = wait.until { driver.find_element(:xpath, "/html/body/app-root/app-forms/section[1]/div/div/div[1]/div/div/form/div[5]/div[2]/div/div/div/select") }
+# dropdown_list = wait.until { driver.find_element(:tag_name, "select") }
+
+print dropdown_list.text
```

Comment:
A better approach is to use 

`dropdown = Selenium::WebDriver::Support::Select.new(dropdown_xpath)
`
You should make use of the functions mentioned here: https://www.selenium.dev/selenium/docs/api/rb/Selenium/WebDriver/Support/Select.html